card/src/main/java/com/adyen/checkout/card/api/PublicKeyConnection.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2021 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by caiof on 5/1/2021.
+ */
+
+package com.adyen.checkout.card.api
+
+import com.adyen.checkout.components.api.SuspendConnection
+import com.adyen.checkout.core.api.Environment
+import com.adyen.checkout.core.log.LogUtil
+import com.adyen.checkout.core.log.Logger
+import org.json.JSONException
+import org.json.JSONObject
+import java.io.IOException
+
+private val TAG = LogUtil.getTag()
+private const val ENDPOINT = "v1/clientKeys/"
+private const val PUBLIC_KEY_JSON_KEY = "publicKey"
+
+class PublicKeyConnection(environment: Environment, clientKey: String) : SuspendConnection<String>(
+    "${environment.baseUrl}$ENDPOINT$clientKey"
+) {
+    @Throws(IOException::class, JSONException::class)
+    override fun call(): String {
+        Logger.v(TAG, "call - $url")
+        val result = String(get())
```

Comment:
Minor: `Charsets.UTF_8` will be used by default here, maybe double check if this is the encoding used by the backend too (if you didn't already 😅).

Reply:
Oh yeah it is, but good pointing out, this could slip by unnoticed. Maybe I can set it explicitly anyway.